kola/tests/kubeadm/checksums.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+set -euo pipefail
+
+fail() {
+    IFS=' ' echo "$*" >&2
+    exit 1
+}
+
```

Comment:
I think usage instructions would be good here when `-h` is passed

Reply:
and how to add a new entry (e.g., first copy the map entry and set the new version but leave the wrong checksums, then run it and update the checksums)

Reply:
Sure, will do.

Reply:
Done.

Reply:
Thanks